package.json

```diff
@@ -8,6 +8,7 @@
   "scripts": {
     "bump-gradle-version": "node ./scripts/gradle-version-update.js --package-version=${npm_package_version} && git add app/build.gradle",
     "build": "./gradlew clean assembleDebug && npm run move-apk",
+    "build:win": "gradlew clean assembleDebug && node ./scripts/build-win.js",
```

Comment:
I would also move gradle execution to the build script. You could use the async teen_process.exec call to run it

Reply:
@mykola-mokhnach should the import { exec } from 'teen_process'; come before or after the 'require' call?

Reply:
@mykola-mokhnach I'm getting the following Error, what's wrong here?
`Cannot find package 'teen_process' imported from C:\Source\Git\android-apidemos\scripts\build-win.js`
I added `import { exec } from 'teen_process';` in the build-win.js
and `"teen_process": "^2.0.2"` in the devDependencies

Reply:
ping me in Slack. I think it would be more productive to discuss it there

Reply:
> await tp.exec('gradlew clean assembleDebug')

that's not the correct way to use teen_process's exec function. it should be more like `tp.exec('gradlew', ['clean', 'assembleDebug'])` if i remember correctly